index.coffee

```diff
@@ -150,7 +152,8 @@
       , d.reject
       promise d
     defProp = (obj, name, value) ->
-      Object.defineProperty obj, name, configurable: true, enumerable: false, value: value
+      Object.defineProperty obj, name, configurable: true, enumerable: false, writable: true, value: value
+    postEnrich = (obj) -> obj;
```

Comment:
should be `(obj) -> return` like at line 113, no?

Reply:
I see that the original enrich function returns the obj, so I made the postEnrich function to handle the obj and return right away. The compiled JavaScript (enrich function) would be
```
return postEnrich(obj);
```
However, I can make the postEnrich function to return nothing and the compiled JavaScript  (enrich function) would be
```
onBind(obj);
return obj;

```

Reply:
As the goal of this hook is not to let the user replace the obj (it would make no sense), there is no point in forcing the user to return the object. And for symmetry with the other hook function it should return nothing.